src/pay/batch_payer.py

```diff
@@ -432,6 +444,8 @@ def attempt_single_batch(self, payment_records, op_counter, dry_run=None):
                 payment_item.paid = PaymentStatus.DONE
```

Comment:
```suggestion
                payment_item.paid = PaymentStatus.AVOIDED
```

Reply:
There was a discussion before concerning this and it was agreed back then to leave it to done so that the user is not confused. I would not change it in this PR, but you could open an issue to discuss this if you would like to.